File: baekjoon/단계별로 풀어보기/1차원 배열/4344_평균은 넘겠지.py
# https://www.acmicpc.net/problem/4344

n = int(input())
for _ in range(n):
    scores = list(map(int, input().split()))[1:]
    mean = sum(scores) / len(scores)
    i = 0
    for score in scores:
        if score > mean:
            i += 1
    print('{:.3f}%'.format(round(i / len(scores) * 100, 3)))

File: C#/Baekjoon/심화1/4344_평균은 넘겠지.cs
//https://www.acmicpc.net/problem/4344

int n = int.Parse(Console.ReadLine());
for (int i = 0; i < n; i++)
{
    float[] score = Array.ConvertAll(Console.ReadLine().Split(), float.Parse).Skip(1).ToArray();
    float mean = score.Sum() / score.Length;
    Console.WriteLine("{0:F3}%", (float)score.Count(x => x > mean) / score.Length * 100);
}